### ISSUE TITLE
导出的 h5 文件的三个numpy数组维数问题

### ISSUE THREAD

**Issue by maple123xx** [ypwhs/dogs_vs_cats#19]
你好，首先感谢您的代码。我读了您的代码有很大启发。但是我自己运行除了问题。您的gap.ipynb最后导出的 h5 文件包括三个 numpy 数组：train (25000, 2048)，test (12500, 2048)，label (25000,)。而我得到的是train(399880,2048)，test(199820,2048)，label(25000,)。请问我是哪里出了问题。请求解答，谢谢！

**Comment by ypwhs** [comment]
版本问题，最新版 keras 的 predict_generator 第二个参数已经改为 steps，参考链接：https://keras.io/zh/models/model/#predict_generator
